english/ghostbusters/ghostbusters.md

```diff
@@ -44,7 +44,7 @@ and sparkle as your body and mind are sent reeling.
 
 You awaken to bluriness and a numbing headache as you wonder
 
-WHERE AM I ???
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.